### PR TITLE
Issue #35/#36

### PR DESCRIPTION
**Pull request by SarahNox**
updating name and/or email of user and validating it by entering current password

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -28,6 +28,7 @@
       </div>
     </header>
     <div class="container">
+            <%= flash.inspect %>
```
Comment (sharnik) [inline app/views/layouts/application.html.erb]: This one was jsut for debugging, you should remove it.

File: app/models/user.rb [hunk 2]
```diff
@@ -8,7 +8,9 @@ class User < ActiveRecord::Base
 	format: { with: VALID_EMAIL_REGEX },
 	uniqueness: { case_sensitive: false }
 	has_secure_password
-    validates :password, presence: true, length: { :within => 6..20}, confirmation: true, :on => :update
+    #validates :password, presence: true, length: { :within => 6..20}, :on => :update, confirmation: true
+    #validates_presence_of :password_confirmation, :if => :password_changed?
+    #validate :verify_password
```
Comment (sharnik) [inline app/models/user.rb]: You shouldn't leave a commented-out code in the repository, unless for a really good reason.

**Comment by sharnik** [comment]
Next steps:

1. Remove the debug code and the commented out stuff.
2. Merge master into the branch and fix the conflicts.
3. Merge the Pull Request.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (SarahNox) [inline app/views/layouts/application.html.erb]: done

**Pull request merged**